### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by cgward00** [keplergl/kepler.gl#2006]
**Describe the bug**
Attempts to Export as Image on Kepler, Doesn't Download when Download Button in Pressed.

**To Reproduce**
1. Enter Data to Kepler
2. Click on Export Button
3. Click on Export Image
4. Adjust Export Setting (Multiple Combinations Attempted)
5. Click on Download

**Expected behavior**
Expects Image File to be Downloaded when Download Button is Clicked. 

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome

**Smartphone (please complete the following information):**
 - Version 106.0.5249.103 (Official Build) (64-bit)

**Additional context**
N/A

**Comment by Barathwaja** [comment]
Yes, it's error because **exportImage** was not defined. Attached screenshot error for reference.

![error-keplr](https://user-images.githubusercontent.com/5066436/196762812-bcbeab3f-d0cb-44cc-8a45-5ef9d579c04a.png)

**Comment by yaikohi** [comment]
Duplicate of https://github.com/keplergl/kepler.gl/issues/1960 - I suggest to close this issue to handle it over there?

**Comment by chrisgervang** [comment]
I believe this image export issue is fixed now with #2368. Try `3.0.0-alpha.1`
